x-pack/plugins/ml/public/explorer/explorer_charts/legacy_utils.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import $ from 'jquery';
```

Comment:
is it worth using `querySelector` rather than jquery for tasks like this? just to avoid the extra included code.

Reply:
Done in 85d1567592.